src/Zicht/Bundle/UrlBundle/Command/ImportUrlAliasesCommand.php

```diff
@@ -0,0 +1,201 @@
+<?php
+/**
+ * @author Boudewijn Schoon <boudewijn@zicht.nl>
+ * @copyright Zicht Online <http://zicht.nl>
+ */
+
+namespace Zicht\Bundle\UrlBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Zicht\Bundle\UrlBundle\Aliasing\Aliasing;
+use Zicht\Bundle\UrlBundle\Entity\UrlAlias;
+
+/**
+ * Class ImportUrlAliasesCommand
+ *
+ * @package Zicht\Bundle\UrlBundle\Command
+ */
+class ImportUrlAliasesCommand extends ContainerAwareCommand
+{
+    private $redirectTypeMapping = [
```

Comment:
Assuming you mean, why do `'301'` and `'move'` do the same thing?  I believe it is more convenient.